### PR TITLE
Card name

### PR DESCRIPTION
**Pull request by libinyang**
The patchset does:
1. fix the getting card short name issue
2. provide the card long name inferface

### CODE REVIEW

File: tools/sof-dump-status.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def _loadACard(self):
         for idx in range(0, len(output), 2):
             card_info={}
             card_info['id'] = output[idx].strip().split(' ')[0]
-            card_info['short'] = output[idx].split(' ')[-1].strip()
+            card_info['short'] = output[idx].split(' - ')[-1].strip()
```
Comment (xiulipan) [inline tools/sof-dump-status.py]: A quick question here. is the spliter **space_dash_space** here? it looks very like to '-'
Please clarify. If there is an example is better.

File: tools/sof-dump-status.py [hunk 2]
```diff
@@ -100,7 +100,7 @@ def _loadACard(self):
             card_info['id'] = output[idx].strip().split(' ')[0]
             card_info['short'] = output[idx].split(' - ')[-1].strip()
             card_info['type']=output[idx].strip().split()[1][1:]
-            card_info['desc']=output[idx + 1].strip()
+            card_info['longname']=output[idx + 1].strip()
```
Comment (xiulipan) [inline tools/sof-dump-status.py]: Do you mean the old description is actually the longname?
I remember there is some requirement for alsalib to support longname? Do we need to check the alsalib here first?

File: tools/sof-dump-status.py (hunk 1 above)
Comment (Bin-QA) [reply tools/sof-dump-status.py]: so what is different between those change? you find the short name with the 'space' ?

Comment (libinyang) [reply tools/sof-dump-status.py]: @xiulipan @Bin-QA 
Yes, it is space_dash_space. Our SOF is ok for space only because we don't use any space in our short card name. But as the test case is targeted to be used widely, we should consider some corner cases. I found this bug when I try to run it on my legacy hda driver.

File: tools/sof-dump-status.py (hunk 2 above)
Comment (libinyang) [reply tools/sof-dump-status.py]: OK. I will check it later

**Comment by libinyang** [comment]
checked. It is longname. You can refer sound/core/init.c for details.

**Comment by xiulipan** [comment]
> I can accept long name, but I couldn't understand why need to use ' - ' instead ' '
I think @libinyang have explained this with 

> I found this bug when I try to run it on my legacy hda driver.

@Bin-QA Please re-check and approval if you think this PR is OK.
